.github/actions/setup-dfx/action.yml

```diff
@@ -13,19 +13,10 @@ runs:
         path: |
           /usr/local/bin/dfx
         key: dfx-cache-${{ inputs.dfx-version }}-5
-    - name: Install DFX
-      shell: bash
-      run: |
-        if command -v dfx
-        then
-          echo "DFX restored from cache"
-        else
-          echo "DFX not restored from cache, running install script:"
-          DFX_VERSION=${{ inputs.dfx-version }} sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
-        fi
```

Comment:
Will the install still be skipped if `dfx` has been retrieved from the cache? I can't convinced myself of that look at the action code [(link)](https://github.com/dfinity/setup-dfx/blob/ae91e0993386b99aae546579a4457a74224b4251/action.yml#L16-L22)